package.json

```diff
@@ -86,14 +86,14 @@
     "spec": "cross-env BABEL_ENV=cjs mocha --require @babel/register --reporter spec",
     "test": "npm run spec && npm run lint"
   },
-  "dependencies": {},
   "devDependencies": {
     "@babel/cli": "^7.4.4",
     "@babel/core": "^7.4.5",
     "@babel/preset-env": "^7.7.4",
     "@babel/register": "^7.4.4",
     "@babel/types": "^7.4.4",
     "@rollup/plugin-babel": "^5.0.4",
+    "babel-eslint": "^10.1.0",
```

Comment:
I played with the `.eslintrc` config and found that we don't need a plugin to fix the `s` regex flag issue. This seems to work fine too:

```none
  "parserOptions": {
    "sourceType": "module",
    "ecmaVersion": 2018 <-- HERE
  },
```